go.mod

```diff
@@ -5,6 +5,8 @@ require (
 	github.com/bgentry/speakeasy v0.1.0
 	github.com/btcsuite/btcd v0.20.1-beta
 	github.com/btcsuite/btcutil v1.0.2
+	github.com/confio/ics23-iavl v0.6.0
```

Comment:
As a follow up to this (sometime before tagging the 0.39 sdk), it would be nice to merge this ics23-iavl code into tendermint/iavl, so this can be optimized and integrated in the tendermint codebase.

Do you want to make an issue for that if you agree (I feel odd doing so and cannot label/milestone it)